src/index.tsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
+import { ThemeProvider } from 'emotion-theming'
 import App from 'App'
+import theme from 'theme'
+
+const rootEl = document.getElementById('root')
+
+ReactDOM.render(
+  <ThemeProvider theme={theme}>
```

Comment:
I think it may be more helpful to keep all of our Providers in `App.tsx` rather than spread between here and there.  I don't _think_ that the order in there matters too much -- maybe as the innermost right around the `<Router />`?